kubernetes/config/kube_config.c

```diff
@@ -111,19 +111,18 @@ static char *getWorkingConfigFile(const char *configFileNamePassedIn)
         configFileName = strdup(configFileNamePassedIn);
     } else {
 
-#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__) || defined(__APPLE__) || !defined(HAVE_SECURE_GETENV)
```

Comment:
Please leave this `#if` and nest your `#if` inside of it.

Reply:
I don't think that that would work. Checking for the operating system is not enough. A much more reliable solution is to check whether the symbol exists in the compilation target. That renders the #if above redundant.